jest.config.js

```diff
@@ -72,5 +79,6 @@ module.exports = {
       globalSetup: './config/jest/browser/open.mjs',
       globalTeardown: './config/jest/browser/close.mjs'
     }
-  ]
+  ],
+  reporters: ['default', 'github-actions']
```

Comment:
If I'm reading this right, this'll surface test failures as annotations on PRs? 🙌🏻 

If so, that sounds like an awesome change but it's 'snuck in' – it's not related to the other code coverage changes, and it's not mentioned in the PR description or the commit message.

Can we split it out into a separate commit and make sure it's mentioned in the PR description?

Reply:
@36degrees Done 👍